src/sensors/fake_imu.cpp

```diff
@@ -234,8 +240,11 @@ void FakeImuFromFile::readDataFromFile(std::string acc_file_path,
       value[1] = 0.0;
       value[2] = 9.8;
 
-      val_read->push_back(addNoiseToData(value, noise_));
       // TODO(jack): Add random point of failure in run for now everything works
+      if (is_fail_ && i == 0) {     // fail during acceleration
```

Comment:
This makes the IMU fail all the time. I thought we were trying to do in a specific state? Have things changed? I notice the TODO but doing this means you can't fail at a different time.